101-synapse-poc/scripts/getobjectid.ps1

```diff
@@ -0,0 +1,16 @@
+try {
+    Import-Module AzureAD
```

Comment:
Why not just use the Azure cmdlets since a user deploying this template likely already has them installed?

Get-AzAdUser